modules/ROOT/pages/migrate-from-an-earlier-version-of-bonita.adoc

```diff
@@ -267,14 +267,14 @@ When done, push the updated configuration into the database:
 . If you have done specific configuration and customization in your server original version, re-do it by configuring the application server at `bonita-target-version/server` (or `bonita-target-version` if target version is 7.3.n): customization, libs etc.
 . *If your Bonita version is 7.4 or above before migrating, you can skip this point.* +++<a id="compound-permission-migration">++++++</a>+++
  In the case where deployed resources have required dedicated  xref:resource-management.adoc#permissions[authorizations to use the REST API], these authorizations are not automatically migrated.
- Some manual operations have to be done on files that are  located in the extracted `platform_conf/current` folder (see  xref:bonita-bpm-platform-setup.adoc#update_platform_conf[Update Bonita Platform configuration] for more information). You need to:
+ Some manual operations have to be done on files that are  located in the extracted `platform_conf/current` folder (see  xref:bonita-platform-setup.adoc#update_platform_conf[Update Bonita Platform configuration] for more information). You need to:
  ** Perform a diff between the version before migration and the version after migration of `tenants/[TENANT_ID]/conf/compound-permissions-mapping.properties` and put the additional lines into the file `tenants/[TENANT_ID]/conf/compound-permissions-mapping-custom.properties`
  ** Perform a diff between the version before migration and the version after migration of `tenants/[TENANT_ID]/conf/resources-permissions-mapping.properties` and put the additional lines into the file `tenants/[TENANT_ID]/conf/resources-permissions-mapping-custom.properties`
  ** Perform a diff between the version before migration and the version after migration of `tenants/[TENANT_ID]/conf/dynamic-permissions-checks.properties` and put the additional lines into the file `tenants/[TENANT_ID]/conf/dynamic-permissions-checks-custom.properties`
  ** Report all the content of the version before migration of``tenants/[TENANT_ID]/conf/custom-permissions-mapping.properties`` into the new version.
 . Configure License:
 +
-you need to put a new license in the database: see  xref:bonita-bpm-platform-setup.adoc#update_platform_conf[Platform configuration] for further details.
+you need to put a new license in the database: see  xref:bonita-platform-setup.adoc#update_platform_conf[Platform configuration] for further details.
  There is below a Linux example:
```

Comment:
There is below a Linux example: => Below is a Linux exemple:

Reply:
Done